hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/cluster/SparkExecuteClusteringCommitActionExecutor.java

```diff
@@ -105,6 +106,19 @@ public SparkExecuteClusteringCommitActionExecutor(HoodieEngineContext context,
     validateWriteResult(writeMetadata);
     commitOnAutoCommit(writeMetadata);
     if (!writeMetadata.getCommitMetadata().isPresent()) {
+      WriteOperationType realOperationType = operationType;
+      Option<Map<String, String>> realExtraMetadata = extraMetadata;
+      if (config.getClusteringExecutionStrategyClass().equals("org.apache.hudi.clustering.SparkZSortAndSizeExecutionStrategy")) {
+        realOperationType = WriteOperationType.ZORDER;
+        Map<String, String> metaDataMap = new HashMap<>();
+        if (extraMetadata.isPresent()) {
+          metaDataMap = extraMetadata.get();
+        }
+        metaDataMap.put("_hoodie.zorder.cols", clusteringPlan.getStrategy()
+            .getStrategyParams().getOrDefault("hoodie.clustering.plan.strategy.sort.columns", ""));
```

Comment:
here why use clustering sort columns instead of defining hoodie.zorder.columns? is hoodie.zorder.cols an internal concept?

Reply:
yes hoodie.zorder.cols is used internal.   may be we need not define a new parameter